cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,176 @@
+                .should("be.enabled")
+                .click();
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .should("have.value", updated_instructor_data.phoneNumber);    
+            cy.get("[data-cy=basicInfo-nextButton]")
+                .should("be.enabled")
+                .click();   
+        }); 
+    });
+
+    it("Can enter data properly into the second form and go back to and from the first form with all data intact", () => {
+        cy.fixture("users.json").then(({updated_instructor_data}) => {
+            cy.get("[data-cy=backButton]")
+                .should("be.enabled")
+                .click();
+            cy.get("[data-cy=basicInfo-nextButton]")
+                .should("be.enabled")
+                .click();            
+            cy.get("[data-cy=experience-experience-input")
+                .clear()
+                .should("be.empty")
+                .fastType(updated_instructor_data.experience)
+                .should("have.value", updated_instructor_data.experience);
+            cy.get("[data-cy=experience-biography-input")
+                .clear()
+                .should("be.empty")
+                .fastType(updated_instructor_data.biography)
+                .should("have.value", updated_instructor_data.biography);
+            cy.get("[data-cy=experience-language-input]")
+                .clear()
+                .should("be.empty")
+                .fastType(updated_instructor_data.language)
+                .should("have.value", updated_instructor_data.language)
+                .parent()
+                .siblings("label")
+                .should("have.css", "color", "rgb(67, 181, 217)");
+            cy.get("[data-cy=submitButton]")
+                .should("be.enabled");
+        });
+    });
+
+    it("Should verify if the data from the user on form 1 still persists if we accidently click the back button", () => {
+        cy.fixture("users.json").then(({updated_instructor_data}) => {
+            cy.get("[data-cy=backButton]")
+                .should("be.enabled")
+                .click();
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .should("have.value", updated_instructor_data.phoneNumber);    
+            cy.get("[data-cy=basicInfo-nextButton]")
+                .should("be.enabled")
+                .click();   
+        }); 
+    });
+
+    it("Should simulate a miss click on the back button", () => {
+            cy.get("[data-cy=backButton]")
+                .should("be.enabled")
+                .click();
+            cy.get("[data-cy=basicInfo-nextButton]")
+                .should("be.enabled")
+                .click();   
+    });
+
+    it("Should verify if the data from the user on form 1 still persists if we accidently click the back button", () => {
+        cy.fixture("users.json").then(({updated_instructor_data}) => {
+            cy.get("[data-cy=experience-experience-input")
```

Comment:
Does this work? Youre missing end brackets for all of these selectors

Reply:
that works but I've changed it